examples/configs/user_settings_wolftpm.h

```diff
@@ -38,91 +38,206 @@
+#define WOLFSSL_AES_NO_UNROLL
 
-/* No threading or file system */
-#define SINGLE_THREADED
-/* File system disable */
-#if 0
-    #define NO_FILESYSTEM
-#endif
+#undef  WOLFSSL_AES_SMALL_TABLES
+#define WOLFSSL_AES_SMALL_TABLES
+
+#undef  HAVE_C___ATOMIC
+#define HAVE_C___ATOMIC 1
+
+#undef  HAVE_THREAD_LS
+#define HAVE_THREAD_LS
+
+#undef  NO_DO178
+#define NO_DO178
+
+#undef  TFM_NO_ASM
+#define TFM_NO_ASM
+
+#undef  WOLFSSL_NO_ASM
+#define WOLFSSL_NO_ASM
+
+#undef  WOLFSSL_X86_64_BUILD
+#define WOLFSSL_X86_64_BUILD
 
-/* Enable crypto callbacks */
-#define WOLF_CRYPTO_CB
+#undef  SINGLE_THREADED
+#define SINGLE_THREADED
 
-/* Enable PRNG (SHA2-256) */
-#define HAVE_HASHDRBG
+#undef  WC_NO_RNG
+#define WC_NO_RNG
 
-/* Enable SP math all (sp_int.c) with multi-precision support */
-#define WOLFSSL_SP_MATH_ALL
+#undef  ERROR_QUEUE_PER_THREAD
+#define ERROR_QUEUE_PER_THREAD
 
-/* Enable hardening (timing resistance) */
+#undef  TFM_TIMING_RESISTANT
 #define TFM_TIMING_RESISTANT
+
+#undef  ECC_TIMING_RESISTANT
 #define ECC_TIMING_RESISTANT
-#define WC_RSA_BLINDING
 
-/* Asymmetric */
-#define HAVE_ECC
-#undef NO_RSA
-#define NO_DH
-#ifndef NO_DH
-    #define HAVE_FFDHE_2048
-    #define HAVE_DH_DEFAULT_PARAMS
-#endif
+#undef  NO_SESSION_CACHE
+#define NO_SESSION_CACHE
+
+#undef  RSA_LOW_MEM
+#define RSA_LOW_MEM
+
+#undef  GCM_SMALL
+#define GCM_SMALL
+
+#undef  CURVE25519_SMALL
+#define CURVE25519_SMALL
+
+#undef  ED25519_SMALL
+#define ED25519_SMALL
+
+#undef  WOLFSSL_SMALL_CERT_VERIFY
+#define WOLFSSL_SMALL_CERT_VERIFY
 
-/* Symmetric Hash */
-#undef NO_SHA
-#undef NO_SHA256
-#define WOLFSSL_SHA512
-#define WOLFSSL_SHA384
+#undef  WOLFSSL_NO_ASYNC_IO
+#define WOLFSSL_NO_ASYNC_IO
 
-/* Symmetric Cipher */
-#define HAVE_AES_KEYWRAP
-#define WOLFSSL_AES_DIRECT
+#undef  USE_SLOW_SHA
+#define USE_SLOW_SHA
+
+#undef  USE_SLOW_SHA256
+#define USE_SLOW_SHA256
+
+#undef  USE_SLOW_SHA512
+#define USE_SLOW_SHA512
+
+#undef  WOLFSSL_AES_CFB
 #define WOLFSSL_AES_CFB
-#define HAVE_AESGCM
-#define GCM_TABLE_4BIT
 
-#if 0
-    #define HAVE_POLY1305
-    #define HAVE_CHACHA
-#endif
+#undef  WOLFSSL_USE_ALIGN
+#define WOLFSSL_USE_ALIGN
+
+#undef  NO_DSA
+#define NO_DSA
+
+#undef  NO_ERROR_STRINGS
+#define NO_ERROR_STRINGS
+
+#undef  NO_OLD_TLS
+#define NO_OLD_TLS
+
+#undef  WOLFSSL_NO_TLS12
+#define WOLFSSL_NO_TLS12
+
+#undef  NO_OLD_TLS
+#define NO_OLD_TLS
+
+#undef  NO_WOLFSSL_MEMORY
+#define NO_WOLFSSL_MEMORY
+
+#undef  NO_RSA
+#define NO_RSA
+
+#undef  NO_DH
+#define NO_DH
+
+#undef  NO_ASN
+#define NO_ASN
+
+#undef  NO_ASN_CRYPT
+#define NO_ASN_CRYPT
+
+#undef  NO_BIG_INT
+#define NO_BIG_INT
 
-/* Features */
-#define WOLFSSL_CERT_GEN
-#define WOLFSSL_CERT_REQ
-#define WOLFSSL_CERT_EXT
+#undef  NO_CODING
+#define NO_CODING
 
-#define HAVE_PKCS7
-#define HAVE_X963_KDF
+#undef  WOLFSSL_BASE64_ENCODE
 #define WOLFSSL_BASE64_ENCODE
 
+#undef  NO_SHA
+#define NO_SHA
 
-/* Disables */
-#define NO_DSA
-#define NO_DES3
+#undef  NO_OLD_TLS
+#define NO_OLD_TLS
```

Comment:
You now have duplicates. If you used the options.h directly this file will need cleaned up.